com.github.najepaliya.kleaner.json

```diff
@@ -0,0 +1,110 @@
+{
+    "id": "com.github.najepaliya.kleaner",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kleaner",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host"
```

Comment:
this is a very broad filesystem permission.

Reply:
What would be the suggested change? Only $XDG_PICTURES_DIR?

Reply:
since it's a Qt app (KDE?) doesn't it use portals? (it's supposedly automatic)

Reply:
I have gone ahead and made the suggested changes, please review again.